packages/skin/src/sass/number-input/number-input.scss

```diff
@@ -0,0 +1,55 @@
+.number-input.textbox {
+    background: transparent;
+}
+
+.number-input label:first-child {
+    flex-grow: 1;
+    padding-left: var(--spacing-200);
+}
+
+.number-input label:first-child ~ input.textbox__control {
+    flex-grow: 0;
+}
+
+.number-input--show-delete .icon-btn.number-input__delete {
```

Comment:
Isn't `.number-input--show-delete .icon-btn.number-input__delete` and `.number-input--show-delete .number-input__delete` the same? Should we put with comma instead? same for `.number-input--show-delete .number-input__decrement`

Reply:
I believe the extra specificity is needed but i can check. 
I'll update if not